### DISCUSSION TITLE
How to add more border width to the selected shape with MapboxGL.FillLayer component

### DISCUSSION THREAD

**Discussion by Dev-mahediali** [rnmapbox/maps#2899] [Q&A]
![Screenshot 2023-06-06 at 5 23 42 PM](https://github.com/rnmapbox/maps/assets/121562212/ef47b5be-5d7d-4a93-aa31-a12d6854a923)
![Screenshot 2023-06-06 at 5 21 21 PM](https://github.com/rnmapbox/maps/assets/121562212/350d339e-3c76-4247-9594-d89217108643)

![image](https://github.com/rnmapbox/maps/assets/121562212/11179be2-2a12-4c90-a8c6-e9b64955be6e)
**I want to make this type of selected shape border (white border)**

**Comment by W1MMER** [comment]
You simply add a `<MapboxGL.LineLayer />` layer inside the `MapboxGL.ShapeSource`.

Here's an example:
```js
<MapboxGL.ShapeSource
  id="selectedPolygon"
  shape={standData || EMPTY_FEATURE_COLLECTION}
>
  <MapboxGL.FillLayer
    id="selectedPolygonStroke"
    aboveLayerID="polygonFill"
    minZoomLevel={POLYGON_ZOOM_MINIMUM - 1}
    style={{
      fillAntialias: true,
      fillOpacity: 0.5,
      fillColor: "#ff0000",
    }}
  />
  <MapboxGL.LineLayer
    id="selectedPolygonLine"
    aboveLayerID="polygonFill"
    visibility={isSelected ? 'visible' : 'none'}
    style={{
      lineColor: "#ff0000",
      lineWidth: 2,
    }}
  />
</MapboxGL.ShapeSource>
```
